### ISSUE TITLE
Add support for @change on q-select

### ISSUE THREAD

**Issue by averri** [quasarframework/quasar#6867]
**Is your feature request related to a problem? Please describe.**
I would like to listen for changes in the `q-select` element.

Example: in the initial state, the code below will render the `q-select` with the `English` selected. If a user select `English` again, the message `Value has changed.` will be displayed, which is the wrong semantic.

```html
<div id="q-app">
  <div class="row q-pa-lg">
 <q-select label="Locale" class="col-4"
                  v-model="locale" 
                  @input="onChange"
                  :options="localeOptions"
                  map-options emit-value
                  :multiple="false"/>
  </div>
</div>
```
```javascript 
new Vue({
  el: '#q-app',
  data: function () {
    return {
      locale: 'en-US',
      localeOptions: [
        {label: 'Portuguese', value: 'pt-BR'},
         {label: 'English', value: 'en-US'}
      ]
    }
  },
  methods: {
  	onChange: function () {
      this.$q.notify('Value has changed.')
    }
	}
})
```

**Describe the solution you'd like**
I would like to have the `@change` support in the `q-select`, and other components that are missing it. The semantic should be the same as the `@change` support in the component `q-input`.

**Describe alternatives you've considered**
A possible workaround is to create watchers. But this is not a good solution when you have to watch too many fields in the same component, or when the fields are in the store.

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.9.16.

**Comment by averri** [comment]
Great work, I see the fix in the v1.9.16.
